src/braille/internal/braille.cpp

```diff
@@ -20,71 +20,59 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
```

Comment:
If we moved the export to a new module, then the old export module is not needed and should it be deleted?

Reply:
I would recommend adding a new module and moving (deleting) the old module into two separate tasks, two separate PRs

Reply:
@igorkorsukov, do you mean like PR https://github.com/musescore/MuseScore/pull/17604? The intention is for that one to get merged first, then I will drop that commit from this PR.

Reply:
It is not very clear to me why to keep the old export module, in fact it has become just a wrapper.
I think it would be better to finish with the new module, merge this PR, and then move (delete) the old one. But I do not insist.

Reply:
I see. I wanted to move the export code first so that I could reduce the size of the diff when adding the new code, and so that I could keep the export tests passing throughout to make sure I didn't break anything.

What I will do is move all the export code in PR #17604, including the tests, so that the entire export module exists in `src/braille`. Then in this PR I will add the new code for the live braille.